core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/RawlsBillingProjectComponent.scala

```diff
@@ -106,9 +106,14 @@ final case class WorkspaceManagerResourceMonitorRecord(jobControlId: UUID,
                                                        jobType: String,
                                                        workspaceId: Option[UUID],
                                                        billingProjectId: Option[String],
-                                                       createdTime: Instant
+                                                       createdTime: Timestamp
 )
 
+object WorkspaceManagerResourceJobType extends Enumeration {
```

Comment:
@blakery if you wish to get into enforcing the use of the enum by adding serialization support and the like, go for it!